Silicon/QemuSocPkg/Library/FirmwareUpdateLib/FirmwareUpdateLib.c

```diff
@@ -285,31 +285,31 @@ GetFirmwareUpdateInfo (
   ASSERT (UpdatePartition != NULL);
   UpdatePartition->RegionCount   = RegionNumber;
 
-  if (CompareGuid(&ImageHdr->UpdateImageTypeId, &gCfgFWUpdateImageFileGuid) == TRUE) {
+  if (CompareGuid(&ImageHdr->UpdateImageTypeId, &gSblCompFWUpdateImageFileGuid) == TRUE) {

```

Comment:
No need to check with TRUE since CompareGuid() returns BOOLEAN.

Reply:
Done